src/components/SharePanel/SharePanel.module.scss

```diff
@@ -0,0 +1,97 @@
+@import "~@santiment-network/ui/variables.scss";
+@import "~@santiment-network/ui/mixins.scss";
+
+.wrapper {
+    min-width: 350px;
```

Comment:
@DmitriVanGuard Default value for mobile devices, and mixin breakpoint value for table and desktop.
For mobile devices, we need to have auto full width.